core/domain/story_domain.py

```diff
@@ -1369,15 +1374,31 @@ def validate(self):
                 'Expected description to be a string, received %s'
                 % self.description)
 
-        if not isinstance(self.node_count, int):
+        if not isinstance(self.node_titles, list):
             raise utils.ValidationError(
-                'Expected node_count to be an int, received \'%s\'' % (
-                    self.node_count))
+                'Expected node_titles to be a list, received \'%s\'' % (
+                    self.node_titles))
```

Comment:
Nit pick: perhaps log type(self.node_titles) here

Reply:
Here and elsewhere

Reply:
This is how validations are done elsewhere in the codebase as well, so not sure if we should change it here. Also, logging the whole thing would help in identifying the type as well, right?

Reply:
You're right, I guess logging it should be enough in most cases

Reply:
Done.